test/e2e/network/dns.go

```diff
@@ -44,9 +44,8 @@ var _ = SIGDescribe("DNS", func() {
 	framework.ConformanceIt("should provide DNS for the cluster ", func() {
 		// All the names we need to be able to resolve.
 		// TODO: Spin up a separate test service and test that dns works for that service.
+		// NOTE: This only contains the FQDN and the Host name, for testing partial name, see the test below
 		namesToResolve := []string{
-			"kubernetes.default",
-			"kubernetes.default.svc",
```

Comment:
/cc @thockin @kubernetes/cncf-conformance-wg @kubernetes/sig-architecture-pr-reviews 
I'm getting similar feels as https://github.com/kubernetes/kubernetes/pull/70156#issuecomment-438586758 about dropping PQDN resolution from conformance, I lack as much context on where that discussion has gone.  Are we OK with this?